Makefile

```diff
@@ -664,3 +664,32 @@ recover_db: run_db
 explorer_fetch_old_batches:
 	@cd explorer && \
 	./scripts/fetch_old_batches.sh 1728056 1729806
+
+__TRACKER__:
+
+tracker_devnet_start: tracker_run_db
+	@cd operator_tracker/ && \
+		cargo run -r -- --env-file .env.dev
+
+tracker_install: tracker_build_db
+	cargo install --path ./operator_tracker
+
+tracker_build_db:
+	@cd operator_tracker && \
+		docker build -t tracker-postgres-image .
+
+tracker_run_db: tracker_build_db tracker_remove_db_container
+	@cd operator_tracker && \
+		docker run -d --name tracker-postgres-container -p 5432:5432 -v tracker-postgres-data:/var/lib/postgresql/data tracker-postgres-image
```

Comment:
5432 is also exposed for the Explorer DB, it will probably bring some issues on stage, but probably not on prod.

Reply:
port was changed to 5433